source/spark_sql_migrations/.gitignore

```
@@ -0,0 +1,8 @@
+# Exclude directory created by wheel creation
+GEHMigrationPackage.egg-info
```

nit: Simplify future maintenance.
```suggestion
*.egg-info
```